README.md

```diff
@@ -68,41 +68,25 @@ Marquez uses a _multi_-project structure and contains the following modules:
 * [`api`](https://github.com/MarquezProject/marquez/tree/main/api): core API used to collect metadata
 * [`web`](https://github.com/MarquezProject/marquez/tree/main/web): web UI used to view metadata
 * [`clients`](https://github.com/MarquezProject/marquez/tree/main/clients): clients that implement the HTTP [API](https://marquezproject.github.io/marquez/openapi.html)
-* [`integrations`](https://github.com/MarquezProject/marquez/tree/main/integrations): integrations with other systems (ex: [`Airflow`](https://github.com/MarquezProject/marquez/tree/main/examples/airflow))
```

Comment:
Maybe worth calling out in the README that the integrations have been deleted and now exist only the OL repo

Reply:
Good point!

Reply:
I did add the update to the changelog, but maybe we can have it in the README.md for a while 👍